<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutLauncher</class>
 <widget class="QWidget" name="AboutLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/res.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPlainTextEdit" name="tText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
background-image: url(:);
border: 0px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="bLicense">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>422</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 60);
border-width:1px;
border-style:solid;
border-radius:4px;
color: rgb(0, 0, 0);
background-image: url(:);</string>
   </property>
   <property name="text">
    <string>License</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bDevelopers">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>422</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 60);
border-width:1px;
border-style:solid;
border-radius:4px;
color: rgb(0, 0, 0);
background-image: url(:);</string>
   </property>
   <property name="text">
    <string>Developers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bTranslators">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>422</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 60);
border-width:1px;
border-style:solid;
border-radius:4px;
color: rgb(0, 0, 0);
background-image: url(:);</string>
   </property>
   <property name="text">
    <string>Translators</string>
   </property>
  </widget>
  <widget class="QLabel" name="lTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>7</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string notr="true">Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-5</y>
     <width>391</width>
     <height>511</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>511</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>511</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/popup.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bClose">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>5</y>
     <width>21</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-style:solid;
border-radius:4px;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>tText</zorder>
  <zorder>bLicense</zorder>
  <zorder>bDevelopers</zorder>
  <zorder>bTranslators</zorder>
  <zorder>lTitle</zorder>
  <zorder>bClose</zorder>
 </widget>
 <resources>
  <include location="../../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
